doc/rst/developer/bestPractices/TestSystem.rst

```diff
@@ -606,13 +608,41 @@ suite(s) in which your graph appears, and you should see data for it.
 (Note that for a new graph with only one day of data, it can be hard
 to see the singleton points at first).
 
+Multilocale Performance Testing
++++++++++++++++++++++++++++++++
+Writing a performance test for multilocale setting is similar to single locale
+counterpart. However, helper file suffixes must be as follows.
+
+
+===================== ==================== 
+ Single Locale         Multilocale         
+===================== ==================== 
+ ``.numlocales``       ``.ml-numlocales``  
+ ``.perfexecopts``     ``.ml-execopts``    
+ ``.perfcompopts``     ``.ml-compopts``    
+ ``.perfkeys``         ``.ml-keys``        
+ ``.graph``            ``.ml-perf.graph``  
+ ``.execenv``          ``.ml-execenv``     
+===================== ==================== 
+
+Graph files for multilocale performance tests are listed in ``ML-GRAPHFILES``
+instead of ``GRAPHFILES``.
+
+Finally to run a multilocale performance test ``start_test --perflabel ml-``
+must be used.
+
+Multilocale Communication Counts Testing
+++++++++++++++++++++++++++++++++++++++++
+Writing a communication counts test for multilocale setting is similar to the
+performance counterpart. However, for helper files ``cc-`` label is used instead
+of ``ml-``.
+
 Test Your Test Before Submitting
 ++++++++++++++++++++++++++++++++
 
-Before submitting your test for review, be sure that it works under
-both ``start_test`` and ``start_test -performance`` modes when running
-within the directory (or directories) in question.  Nothing is more
-embarrassing than committing a test that doesn't work on day one.
+Before submitting your test for review, be sure that it works with relevant
+``start_test`` command.  Nothing is more embarrassing than committing a test
```

Comment:
I'm not sure why you removed the `-performance` portion?  This section is under performance testing specifically and was written to point out to users that both strategies should work (since we've had folks fail to check one or the other)

Reply:
It would be fair to have a version under normal testing that doesn't list both, but I wouldn't remove this one

Reply:
I had removed it to avoid listing 4 possible options (with ml and cc). I just added a bullet list to include everything.